assignments/models.py

```diff
@@ -121,3 +121,30 @@ class Meta:
 
     def __str__(self):
         return self.name
+
+
+class Student(models.Model):
```

Comment:
I don't know how you have normally done this, but are there not going to be any translations at all in this project? I don't see any `ugettext_lazy` imports anywhere.